packages/lwc/scripts/utils/generate_targets.js

```diff
@@ -21,15 +21,21 @@ module.exports = function generateTargets(targets, opts = {}) {
     let jobsCompleted = 0;
 
     return new Promise((resolve, reject) => {
+        let overallError;
         targets.forEach((config) => {
             workers(config, (err) => {
                 if (err) {
-                    return reject(err);
+                    console.error(err);
```

Comment:
would this error be displayed 2 times? (here and in in the [catch](https://github.com/salesforce/lwc/pull/2434/files#diff-44fa6b5b7ea9a7e00170db776bb14c86cf1ff3ce17bfb1e7eccf6976a47fa26aR93) )
What happens when there are multiple errors? e.g. multiple targets generate errors.

Reply:
Yes I think it would be displayed two times. I figured it was easiest to just log all the exceptions, since as you say there could be multiple.

Reply:
In the case of multiple exceptions we print them all but then `reject` with the last one.

Reply:
I figured. wouldn't be better to reject with an array of exceptions and print them all, only once in the catch?

Reply:
I thought it was bad practice to reject with anything but an `Error`? But I guess if we're the only ones handling it, then it's fine.

Reply:
OK, I came up with a much simpler solution using `Promise.all` and `promisify`. No need to do any fancy error handling ourselves; just let `Promise.all` handle it.